internal/common/consts.go

```diff
@@ -47,6 +47,8 @@ const (
 	LvmConfigDir                      = "lvm-configuration"
 	LvmDevicesPath                    = "/etc/lvm/devices/system.devices"
 	CABundleFilePath                  = "/etc/pki/ca-trust/extracted/pem/tls-ca-bundle.pem"
+
+	LcaNamespace = "openshift-lifecycle-agent"
```

Comment:
There's also `LcaPrecacheNamespace` ...do you mind removing that one just use this one?

Reply:
+1